tlsh.go

```diff
@@ -285,11 +340,31 @@ func HashBytes(blob []byte) (hash string, err error) {
 
 //Hash calculates the TLSH for the input file
 func Hash(filename string) (hash string, err error) {
-	f, err := os.Open(filename)
+	hash, err = hashFile(filename)
+	if err != nil {
+		return "", err
+	}
+
+	return hash, nil
+}
+
+// Diff calculate distance between two files
+func Diff(filenameA, filenameB string) (int, error) {
```

Comment:
IMO we fix the API with the 1.0.0 release. I was thinking about a "feature complete" 0.2.x release (merging this PR) and then after some polishing the 1.x release. Good idea to use `hash` as a guideline, will have a look and will add some doc in the wiki.